experiment/process-e2e-suite-log/README.md

```diff
@@ -0,0 +1,63 @@
+# target 
+To process the input file and extract only a given selection of visited endpoints.
+
+## "input file":
+it is a "master log," that is the collection of the endpoints visited while executing a testing run.  
+Here is the [input][input source]: 
+
+This file has an expected and known format: 
```

Comment:
Might be easier to if you put this in a \```blahblahblah\``` block:

```
---
2018-07-31 10:22
sig-description
testcase description
/path/to/testcase.go
GET /not/sure/about/lines/6+
LIST /next/request
...
---
...
```

Reply:
added the example as you mentioned. Thanks!